public/static/docs/command-reference/metrics/show.md

```diff
@@ -26,43 +26,41 @@ The optional `targets` argument can contain several metric files. With the `-R`
 option, a target can even be a directory, so that DVC recursively shows all
 metric files in it.
 
-Providing a `type` (`-t` option) overwrites the full metric specification (both
+Providing a `type` (`-t` option) overrides the full metric specification (both
 `type` and `xpath` fields) defined in the
-[DVC-file](/doc/user-guide/dvc-file-format) (usually set originally with the
-`dvc metrics modify` command).
+[DVC-file](/doc/user-guide/dvc-file-format) (originally with
+`dvc metrics modify`, typically).
 
 If `type` (via `-t`) is not specified and only `xpath` (`-x` option) is, only
-the `xpath` field is overwritten in its DVC-file. (DVC will first try to read
+the `xpath` field from the DVC-file is overridden. (DVC will first try to read
 `type` from the DVC-file, but it can be automatically detected by the file
 extension.)
 
-> Alternatively, see `dvc metrics modify` command to learn how to apply `-t` and
-> `-x` permanently.
+> See `dvc metrics modify` to learn how to apply `-t` and `-x` permanently.
+
+See also `dvc metrics diff` to show changes in metrics between different
+versions of the <abbr>repository</abbr>.
```

Comment:
need to come up with a better message here .. it's not informative - it describes `git diff` more or less

Reply:
Well, its pretty much the short definition of `dvc metrics show`, copied from its cmd ref... But let me think... ⏳

Reply:
How about

```markdown
When metrics change in more recent <abbr>repository</abbr> versions, an
alternative way to display them is `dvc metrics diff`, which shows a table of
the changes.
```
? Updated in 455e7932